### PR TITLE
[FAB-16147] Update Commercial Paper to work with v2

### PR DESCRIPTION
**Pull request by mbwhite**
Update the applications and scripts to use the new v2
SDKs and the new lifecycle

Add in a basic script based on the readme.md that does
a basic run of the scenario

Signed-off-by: Matthew B White <whitemat@uk.ibm.com>

### CODE REVIEW

**Comment by mbwhite** [comment]
Note: this is using the basic-network; have started moving this to test-network as it better represents the scenario,

File: commercial-paper/organization/magnetocorp/contract-java/src/main/java/org/example/ledgerapi/State.java [hunk 1]
```diff
@@ -54,7 +53,7 @@ public static String makeKey(String[] keyParts) {
     }
 
     public static String[] splitKey(String key) {
-        System.out.println("Splitting key " + key + "   " + java.util.Arrays.asList(key.split(":")));
+        System.out.println("Splittin gkey " + key + "   " + java.util.Arrays.asList(key.split(":")));
```
Comment (sstone1) [inline commercial-paper/organization/magnetocorp/contract-java/src/main/java/org/example/ledgerapi/State.java]: I think this was right before 😂

File: commercial-paper/organization/magnetocorp/contract-java/src/main/java/org/example/CommercialPaper.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /*
- *  SPDX-License-Identifier: Apache-2.0
+ * SPDX-License-Identifier:
```
Comment (sstone1) [inline commercial-paper/organization/magnetocorp/contract-java/src/main/java/org/example/CommercialPaper.java]: No license identifier?

File: commercial-paper/organization/magnetocorp/contract-java/src/main/java/org/example/ledgerapi/State.java (hunk 1 above)
Comment (mbwhite) [reply commercial-paper/organization/magnetocorp/contract-java/src/main/java/org/example/ledgerapi/State.java]: 😞  so need a compiler for English. ..

**Pull request merged**